resources/src/commands.rs

```diff
@@ -3,3 +3,4 @@
 // This will allow you to import them into main.rs.
 // Read more here: https://doc.rust-lang.org/rust-by-example/mod.html
 pub mod ping;
+pub mod buns;
```

Comment:
I think mods have to be alphanumeric order so the formatter is going to complain.

Reply:
Figured out how to do it from my terminal and got it solved locally. We should add that to contributing.md

Reply:
For sure. Throw the formatting commands in the `CONTRIBUTING.md` pls.